scan-citrix-netscaler-version.py

```diff
@@ -340,7 +344,7 @@ def scan_netscaler_version(target: str, client: httpx.Client) -> NetScalerVersio
             )
         else:
             error = "No valid data found, probably not a Citrix NetScaler"
-            logging.info("No valid data found, probably not a Citrix NetScaler")
+            raise ValueError(error)
```

Comment:
I prefer to have the error logged in the `NetScalerVersion()` result. The reason why it looks weird is because the error was not logged correctly, see suggestion below to fix this.

I do think the `logging.info()` is redundant, so if you just remove this line it will also be ok.
```suggestion
```